src/renderer/component/speechUpload/view.jsx

```diff
@@ -85,15 +24,12 @@ class SpeechUpload extends React.PureComponent {
         <div
           className="card__content"
           style={
-            this.state.thumbnailUploadStatus !== "manual"
-              ? null
-              : { display: "none" }
+            this.props.uploadStatus !== "manual" ? null : { display: "none" }
```

Comment:
Upload status states should either be locally declared variables (if used only in one file) or exportable constant definitions if used in multiple files.

https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants